### ISSUE TITLE
Weight decay and Momentum implemenation are not included.

### ISSUE THREAD

**Issue by leekyungmoon** [kratzert/finetune_alexnet_with_tensorflow#7]
I got a lot of information to implement alexnet, thank you.

But two implementations are not included in your blog and code but included in alexnet.

Is it intended?

**Comment by kratzert** [comment]
Well lets say it like this: This repository isn't meant to reproduce the paper results of the orignal AlexNet paper (training the entire network from scratch on the ImageNet data). The use of the code of this repository is meant for finetuning AlexNet (only train some of the last layers) and for this, it works pretty well.

Hope this answers your question.
